learn/advanced/working_with_dates.md

```diff
@@ -0,0 +1,84 @@
+# Working with dates
+
+In this guide, you will learn about Meilisearch's approach to date and time values, how to prepare your dataset for indexing, and how to chronologically sort and filter search results.
+
+## Preparing your documents
+
+To filter and sort search results chronologically, your documents must have at least one numeric field containing a [UNIX timestamp](https://kb.narrative.io/what-is-unix-time).
+
+As an example, consider a database of video games. In this dataset, the release year is formatted as a timestamp:
+
+```json
+[
+  {
+    "id": 0,
+    "title": "Return of the Obra Dinn",
+    "genre": "adventure",
+    "release_timestamp": 1538949600
+  },
+  {
+    "id": 1,
+    "title": "The Excavation of Hob's Barrow",
+    "genre": "adventure",
+    "release_timestamp": 1664316000
+  },
+  {
+    "id": 2,
+    "title": "Bayonetta 2",
+    "genre": "action",
+    "release_timestamp": 1411164000
+  }
+]
+```
+
+If your date field is expressed in a format other than a numeric timestamp, like [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html), you must convert it before indexing it with Meilisearch.
+
+Most programming languages have built-in tools to help you with this process. The JavaScript example below converts a game's release date, `"2018-10-18"`, to a numeric timestamp:
+
+```js
+let game = {
+  "id": 0,
+  "title": "Return of the Obra Dinn",
+  "genre": "adventure",
+  "release_date": "2018-10-18T00:00Z"
+};
+
+const timestampInMilliseconds = Date.parse(game.year); // Date.parse returns the timestamp in milliseconds
+const timestamp = timestampInMilliseconds / 1000; // UNIX timestamps must be in seconds
+
+game = {
+  "id": 0,
+  "title": "Return of the Obra Dinn",
+  "genre": "adventure",
+  "release_date": "2018-10-18T00:00Z",
+  "release_timestamp": timestamp
+};
+```
+
+:::tip
+When preparing your dataset, it can be useful to leave the original date and time fields in your documents intact. In the example above, we keep the `release_date` field because it is more readable than the raw `timestamp`.
+:::
+
+After adding a numeric timestamp to all documents, [index your data](/reference/api/documents.md#add-or-replace-documents) as usual. The example below adds a <a id="downloadRestaurants" href="/restaurants.json" download="restaurants.json">videogame dataset</a> to a `games` index:
```

Comment:
```suggestion
After adding a numeric timestamp to all documents, [index your data](/reference/api/documents.md#add-or-replace-documents) as usual. The example below adds a <a id="downloadVideogames" href="/videogames.json" download="videogames.json">videogame dataset</a> to a `games` index:
```

Reply:
oops! That's what happens when I'm on a rush 😓